frontend/app/src/utils/text.ts

```diff
@@ -40,6 +40,42 @@ export const toCapitalCase = (string: string): string =>
     txt => txt.charAt(0).toUpperCase() + txt.slice(1)
   );
 
+/**
+ *
+ * @param {string} value - String to convert
+ * @param {'capitalize' | 'sentence' | 'uppercase' | 'lowercase'} transform
+ * @return {string} - String converted to human-readable case
+ * @example
+ * toHumanReadable('POLYGON_POS', 'sentence'); // Polygon Pos
+ * @example
+ * toHumanReadable('POLYGON_POS'); // POLYGON POS
+ * @example
+ * toHumanReadable('polygon_pos', 'uppercase'); // POLYGON POS
+ */
+export const toHumanReadable = (
```

Comment:
Do we have a test for this utility? if not can you add a test also?

Reply:
I was trying not to get the PR too long, but I can add a test for this.